src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -385,7 +385,8 @@ public Command goToState(ArmState state){
     //Command for autonomous, obstructs routine until arm is at setpoint
     return new FunctionalCommand(
                 () -> setState(state), //Init
-                this::updateMotors, //Execute
+                //this::updateMotors, //Moved to periodic
+                null, //Execute
```

Comment:
```suggestion
                () -> {}, //Execute
```
Null doesn't work here